src/app/components/translate-atttributes/translate-attributes.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="modal-body">
+  <div class="d-flex justify-content-between align-items-center">
+    <h5>Translation Attributes</h5>
+    <!-- Add button -->
+    <button class="btn btn-secondary" (click)="createAttribute()">
+      <i class="fa fa-file"></i> Add new Translation Attribute
+    </button>
+  </div>
+
+  <!-- Loop through items and render -->
+  <div class="translation-attributes">
+    <div
+      class="row md-2"
+      *ngFor="let attr of resource['translated-attributes']"
+    >
+      <div class="col-4">
+        <label for="attr-{{ attr.id }}-key">Key</label>
+        <input
+          id="attr-{{ attr.id }}-key"
+          class="form-control"
+          placeholder="Key"
+          [(ngModel)]="attr.key"
+        />
+      </div>
+      <div class="col-5">
+        <label for="attr-{{ attr.id }}-onesky">Onesky Phrase ID</label>
+        <input
+          id="attr-{{ attr.id }}-onesky"
+          class="form-control"
+          placeholder="Onesky Phrase ID"
+          [(ngModel)]="attr['onesky-phrase-id']"
+        />
+      </div>
+      <div class="col-2">
+        <label for="attr-{{ attr.id }}-required">Required?</label>
+        <input
+          id="attr-{{ attr.id }}-required"
+          type="checkbox"
+          placeholder="Required?"
+          [(ngModel)]="attr.required"
+        />
+      </div>
+      <button (click)="removeAttribute(attr)" class="btn btn-warn col-1">
+        <i class="fa fa-times"></i>
```

Comment:
is there a trash can icon available within font awesome we can use here instead?

Reply:
Added a Trash iconn.